tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
+
+conds_we_care_about = ['auditory/left', 'auditory/right',
+                       'visual/left', 'visual/right']
+epochs.equalize_event_counts(conds_we_care_about)  # this operates in-place
+aud_epochs = epochs['auditory']
+vis_epochs = epochs['visual']
+del raw, epochs  # free up memory
+
+###############################################################################
+# Like :class:`~mne.io.Raw` objects, :class:`~mne.Epochs` objects also have a
+# number of built-in plotting methods. One is :meth:`~mne.Epochs.plot_image`,
+# which shows each epoch as one row of an image map, with color representing
+# signal magnitude; the average evoked response and the sensor location are
+# shown below the image:
+
+aud_epochs.plot_image(picks=['MEG 1332', 'EEG 021'])
+
+###############################################################################
+# Estimating evoked responses
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-#   1. Extract the max value of each epoch
+# Now that we have our conditions in ``aud_epochs`` and ``vis_epochs``, we can
+# get an estimate of evoked responses to auditory versus visual stimuli by
+# averaging together the epochs in each condition. This is as simple as calling
+# the :meth:`~mne.Epochs.average` method on the :class:`~mne.Epochs` object,
+# and then using a function from the :mod:`mne.viz` module to compare the
+# global field power for each sensor type of the two :class:`~mne.Evoked`
+# objects:
 
-max_in_each_epoch = [e.max() for e in epochs['aud_l']]  # doctest:+ELLIPSIS
-print(max_in_each_epoch[:4])  # doctest:+ELLIPSIS
+aud_evoked = aud_epochs.average()
+vis_evoked = vis_epochs.average()
 
-##############################################################################
-# It is also possible to read evoked data stored in a fif file:
-
-evoked_fname = data_path + '/MEG/sample/sample_audvis-ave.fif'
-evoked1 = mne.read_evokeds(
-    evoked_fname, condition='Left Auditory', baseline=(None, 0), proj=True)
+mne.viz.plot_compare_evokeds(dict(auditory=aud_evoked, visual=vis_evoked),
+                             show_legend='upper left',
+                             show_sensors='upper right')
 
-##############################################################################
-# Or another one stored in the same file:
+###############################################################################
+# We can also get a more detailed view of each :class:`~mne.Evoked` object
+# using other plotting methods such as :meth:`~mne.Evoked.plot_joint` or
+# :meth:`~mne.Evoked.plot_topomap`. Here we'll examine just the EEG channels,
+# and see the classic auditory evoked N100-P200 pattern over dorso-frontal
+# electrodes, then plot scalp topographies at some additional arbitrary times:
 
-evoked2 = mne.read_evokeds(
-    evoked_fname, condition='Right Auditory', baseline=(None, 0), proj=True)
+aud_evoked.plot_joint(picks='eeg')
+aud_evoked.plot_topomap(times=[0., 0.08, 0.1, 0.12, 0.2], ch_type='eeg')
```

Comment:
Do you think we could do pick_types('eeg') at the beginning and do the rest of the tutorial using one channel type? It might be simpler. But on the other hand, you may want to show that MNE can handle different channel types ...

Reply:
I've been avoiding early use of `pick_types` on purpose, in hopes of keeping the attention of both MEG and EEG folks.  I also think it's nice to show people that the plotting functions have these `picks` parameters.